examples/pennylane_make_your_first_quantum_circuit.py

```diff
@@ -0,0 +1,189 @@
+*NumPy* which is provided via PennyLane. This allows us to use the
+familiar NumPy functions along with Quantum functions.
+"""
+
+import pennylane as qml
+from pennylane import numpy as np
+
+###############################################################################
+# .. important::
+#
+#     When constructing a hybrid quantum/classical computational model with
+#     PennyLane, it is important to always import NumPy from PennyLane, not
+#     the standard NumPy!
+#
+# Creating a device
+# -------------------
+#
+# A **Quantum Circuit** comprises of quantum **operations** and **wires**
+# that encode quantum bits that the operations can act on.
+#
+# .. admonition:: Definition
+#     :class: defn
+#
+#     In PennyLane, the term quantum ‘device’ refers to a computational
+#     object that can apply quantum operations and return an expectation
+#     value.
+#
+# A device could be a hardware device (such as the IBM QX4, via the
+# PennyLane-PQ plugin) or a software simulator (such as Strawberry Fields,
+# via the PennyLane-SF plugin).
+#
+# PennyLane supports two built-in devices for **discrete variable** and 
+# **continuous variable** quantum computation:
+#
+# * :mod:`default.qubit <pennylane.plugins.default_qubit>`: pure state qubit simulator
+#
+# * :mod:`default.gaussian < pennylane.plugins.default_gaussian>`: Gaussian states simulator
+#
+# Additional devices are supported through plugins - see :ref:`plugin ecosystem <plugins>`
+# for more details.
+#
+# Let’s use the qubit simulator device provided by PennyLane. Devices are
+# loaded in PennyLane via the function :func:`~.pennylane.device`:
+
+dev = qml.device('default.qubit', wires = 1)
+
+##############################################################################
+# We call this function with two arguments:
+#
+# * ``name``: the name of the device to be loaded
+#
+# * ``wires``: the number of subsystems to initialize the device with
+#
+# Here, ``wires=1`` means that a single qubit is initiated.
+#
+# Constructing the QNode
+# ----------------------
+#
+# Now that we have initialized our device, we can begin to construct a
+# :class:`pennylane.QNode <pennylane.qnode.QNode>`.
+#
+# .. admonition:: Definition
+#     :class: defn
+#
+#     QNodes are an abstract encapsulation of a quantum function, described
+#     by a quantum circuit. QNodes are bound to a particular quantum
+#     device, which is used to evaluate expectation values of this circuit.
+#
+# First, we need to define the quantum function that will be evaluated in
+# the QNode:
+
+# let's use PennyLane operator RX - Rotation about the X-axis by angle param
+def circuit(param):
+    qml.RX(param, wires=0)
+    return qml.expval.PauliZ(0)
+
+##############################################################################
+# **NOTE:** the function ``circuit()`` is constructed as if it were any
+# other Python function using Python notation (``def fn(...)``); it
```

Comment:
I was confused if it is this instance or all quantum functions. Better?
```suggestion
# other Python function using Python notation (``def fn(...)``); here, it
```

Reply:
the function `circuit()` is the subject of the sentence. I am not sure about adding "here".  I will wait for Nathan's feedback :)